### PR TITLE
'bugfixes'

### PR DESCRIPTION
**Pull request by RadiantFlash7**
## About The Pull Request
This PR contains bug fixes!

And by fixing bugs, I mean, Glory to the Kaiserin, a weapon to surpass metalgear.
Adds in the Kaiserin- a modified-cyborgified version of the Kaiser, equipped with a minigun. It sprays bullet hell.
Also adds a 1/100th chance for the kaiser event to spawn a Kaiserin instead of a kaiser

## Why It's Good For The Game
The Kaiser needs a good waifu, and it's a neat addition. Can also be used in the upcoming event.
It's a fun challenging fight that requires a lot of cooperation to take down.

## Changelog
```changelog
add:Kaiserin
tweak:Adds kaiserin spawn to kaiser event
iconadd: Adds kaiserin sprites
```

### CODE REVIEW

File: zzzz_modular_occulus/code/modules/mob/living/carbon/superior_animal/roach/kaiserin.dm [hunk 1]
```diff
@@ -0,0 +1,126 @@
+/mob/living/carbon/superior_animal/roach/kaiser/kaiserin
+	name = "Kaiserin Roach"
+	desc = "The empress her self. A weapon to surpass metal gear."
```
Comment (Jamini) [inline zzzz_modular_occulus/code/modules/mob/living/carbon/superior_animal/roach/kaiserin.dm:3]: herself

File: zzzz_modular_occulus/code/modules/mob/living/carbon/superior_animal/roach/kaiserin.dm [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+	melee_damage_upper = 40
+
+
+	wander = FALSE //No more sleepwalking
+	projectilesound = 'zzzz_modular_occulus/sound/weapons/guns/fire/minigun_fire.ogg'
+	projectiletype = /obj/item/projectile/bullet/srifle/nomuzzle
+
+
+
+/mob/living/carbon/superior_animal/roach/kaiser/kaiserin/handle_ai()
```
Comment (Jamini) [inline zzzz_modular_occulus/code/modules/mob/living/carbon/superior_animal/roach/kaiserin.dm:30]: Why is this being replicated? If it is identical to the Kaiser proc, we don't need to re-define it

File: code/game/gamemodes/events/hidden_events/kaiser.dm [hunk 3]
```diff
@@ -49,6 +49,9 @@ It will also bring a hoard of roaches with it.
 		failure = TRUE
 
 /datum/event/kaiser/start()
+	if(prob(1))			// occulus kaiserin start edit
+		spawn_kaiserin()
```
Comment (Jamini) [inline code/game/gamemodes/events/hidden_events/kaiser.dm:53]: Why are we adding a whole new proc here?

File: code/game/gamemodes/events/hidden_events/kaiser.dm [hunk 4]
```diff
@@ -49,6 +49,9 @@ It will also bring a hoard of roaches with it.
...
+		return			// occulus kaiserin end edit
 	spawn_mobs()
 
 /datum/event/kaiser/proc/spawn_mobs()
```
Comment (Jamini) [inline code/game/gamemodes/events/hidden_events/kaiser.dm:57]: When we could just put the mob swap down here with two lines?

File: zzzz_modular_occulus/code/modules/mob/living/carbon/superior_animal/roach/kaiserin.dm [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+			counter--
+		else
+			counter++
+		if(counter > 16)
+			counter = 1
+		if(counter < 1)
+			counter = 16
+		shoot_projectile(start_turf, counter * 22.5)
+
+/datum/event/kaiser/proc/spawn_kaiserin()
```
Comment (Jamini) [inline zzzz_modular_occulus/code/modules/mob/living/carbon/superior_animal/roach/kaiserin.dm:108]: We don't need to duplicate this code. Just modify spawn_mobs instead

File: zzzz_modular_occulus/code/modules/mob/living/carbon/superior_animal/roach/kaiserin.dm (hunk 1 above)
Comment (RadiantFlash7) [reply zzzz_modular_occulus/code/modules/mob/living/carbon/superior_animal/roach/kaiserin.dm:3]: Fixed!

File: zzzz_modular_occulus/code/modules/mob/living/carbon/superior_animal/roach/kaiserin.dm (hunk 2 above)
Comment (RadiantFlash7) [reply zzzz_modular_occulus/code/modules/mob/living/carbon/superior_animal/roach/kaiserin.dm:30]: ...Ah- Yeah. Originally I had it's minigun shooting function from there, but I removed it in favor of other code. Thats my bad.

File: code/game/gamemodes/events/hidden_events/kaiser.dm (hunk 3 above)
Comment (RadiantFlash7) [reply code/game/gamemodes/events/hidden_events/kaiser.dm]: Cause I'm a bad coder, and I glanced at the spawn_mobs proc at 2 am and went 'durrr, I can't adjust that to work.'
Fixed.

**Review by Jamini — APPROVED** [review]
Integration is failing due to another bug. Should be fine.

**Pull request merged**